src/Generator/Generators/HtmlOmdGenerator.cs

```diff
@@ -214,7 +214,6 @@ public void WriteType(INamedTypeSymbol type, INamedTypeSymbol oldType)
                     if (iface.wasRemoved) sw.Write("</span>");
                     i++;
                 }
-                sw.WriteLine("</span>");

```

Comment:
Where's the opening span inside this method?

Reply:
There wasn't one. Which is why I removed this closing span

Reply:
d'oh! Brainfairt. Thanks!